scheduler/test/cook/test/mesos/optimizer.clj

```diff
@@ -0,0 +1,24 @@
+(ns cook.test.mesos.optimizer
+  (:use clojure.test)
+  (:require [clj-time.core :as t]
+            [clojure.core.async :as async]
+            [cook.mesos.optimizer :as optimizer]))
+
+;; Tests to make sure data flows and validates properly
+(deftest test-optimizer-cycle
```

Comment:
This test should have an assertion somewhere, preferably on the return value of `optimizer-cycle!`

Reply:
Added